Assignments/stu10/assignments.py

```diff
@@ -104,3 +104,8 @@ def ex7():
 # Place your functions here...
 #
 
+#ex1
+def sort_people(people_list, x, y):
```

Comment:
Well, we really need to check if the list is asc or desc... so we need an if statement here.